src/contents/blog/2025-05-11-git-delete-squashed-branch/index.md

```
@@ -0,0 +1,117 @@
+
+- [Git で squash merge後にmergeされたコミットを含むbranchがコンフリクトするのを解決する #onto - Qiita](https://qiita.com/nakamasato/items/680f3908437b72eb7186)
+- [squash mergeの環境でCascading PRsでコンフリクトした時 - oinume journal](https://journal.lampetty.net/entry/resolve-squash-merge-conflict)
+
+しかし、マージ済みブランチを削除するとき `-d` ではなく `-D` で消さなければいけない。
+定期的に `git pull --prune` してローカルのマージ済みブランチも掃除しているのでコマンド一発でマージ済みブランチを消せないのは不便に感じる。
+
+## `--no-ff` の場合
+
+Create a merge commit でマージしたときは次の alias を `~/.config/git/config` に設定しているので `git delete-merge-branch` コマンドでマージ済みブランチを削除している。
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)